src/riverroute/RtmTimeManager.F90

```diff
@@ -124,11 +124,12 @@ end subroutine timemgr_setup
 
 !=========================================================================================
 
-subroutine timemgr_init( dtime_in )
+subroutine timemgr_init( dtime_in, curr_date_in )
 
   ! Initialize the ESMF time manager from the sync clock
   !
-  integer, intent(in) :: dtime_in         ! Time-step (sec)
+  integer, intent(in) :: dtime_in               ! ` Time-step (sec)
+  type(ESMF_Time), intent(in), optional :: curr_date_in   !  Current time from coupler``
```

Comment:
This and below doesn't need the extra complexity of being optional. So I'll remove that and retest.

Reply:
Made that simplification so resolving.